messages/compiler/cmfc.py

```diff
@@ -25,13 +25,19 @@ def translate(ast, language, namespace, output):
     if language == "cpp":
         print("Generating C++ source code")
         from cpp import cppgen
-
         header, impl = cppgen.translate(ast, output + ".hpp", namespace)
         with open(output + ".hpp", "w") as f:
             f.write(header)
         with open(output + ".cpp", "w") as f:
             f.write(impl)
 
+    elif language == "python":
+        print("Generating Python source code")
```

Comment:
At some point we should replace all `print` statements with proper logging. No need to change now.